code/modules/gear_presets/survivors/lv_624/corporate_dome_insert_lv624.dm

```diff
@@ -0,0 +1,51 @@
+// /obj/effect/landmark/survivor_spawner/lv624_corporate_dome/cl
+// corporatedomehold.dmm
+
+/datum/equipment_preset/survivor/wy/executive
+	name = "Survivor - LV-624 Paranoid Corporate Liaison"
+	flags = EQUIPMENT_PRESET_START_OF_ROUND
+	paygrade = PAY_SHORT_WYC5
+	skills = /datum/skills/civilian/survivor/manager
+	assignment = "LV-624 Corporate Liaison"
+	idtype = /obj/item/card/id/silver/clearance_badge/cl
+	faction_group = list(FACTION_WY, FACTION_SURVIVOR)
+	access = list(
+		ACCESS_WY_GENERAL,
+		ACCESS_WY_COLONIAL,
+		ACCESS_WY_MEDICAL,
+		ACCESS_WY_SECURITY,
+		ACCESS_WY_RESEARCH,
+		ACCESS_WY_ARMORY,
+		ACCESS_CIVILIAN_PUBLIC,
+		ACCESS_CIVILIAN_RESEARCH,
+		ACCESS_CIVILIAN_ENGINEERING,
+		ACCESS_CIVILIAN_LOGISTICS,
+		ACCESS_CIVILIAN_BRIG,
+		ACCESS_CIVILIAN_MEDBAY,
+		ACCESS_CIVILIAN_COMMAND,
+	)
+	languages = list(LANGUAGE_ENGLISH, LANGUAGE_JAPANESE, LANGUAGE_RUSSIAN)
+
+	survivor_variant = CORPORATE_SURVIVOR
+
+/datum/equipment_preset/survivor/wy/executive/load_gear(mob/living/carbon/human/new_human)
+	new_human.equip_to_slot_or_del(new /obj/item/clothing/under/liaison_suit/field(new_human), WEAR_BODY)
+	if(SSmapping.configs[GROUND_MAP].environment_traits[MAP_COLD])
+		add_ice_colony_survivor_equipment(new_human)
+	new_human.equip_to_slot_or_del(new /obj/item/clothing/head/helmet/marine/veteran/pmc(new_human), WEAR_HEAD)
+	if(new_human.disabilities & NEARSIGHTED)
+		new_human.equip_to_slot_or_del(new /obj/item/clothing/glasses/sunglasses/prescription(new_human), WEAR_EYES)
+	else
+		new_human.equip_to_slot_or_del(new /obj/item/clothing/glasses/sunglasses(new_human), WEAR_EYES)
+	new_human.equip_to_slot_or_del(new /obj/item/clothing/suit/armor/vest(new_human), WEAR_JACKET)
+	new_human.equip_to_slot_or_del(new /obj/item/storage/belt/gun/m4a3/vp78(new_human), WEAR_WAIST)
+	new_human.equip_to_slot_or_del(new /obj/item/device/radio/headset/distress/WY(new_human), WEAR_L_EAR)
+	new_human.equip_to_slot_or_del(new /obj/item/storage/secure/briefcase(new_human), WEAR_L_HAND)
+	new_human.equip_to_slot_or_del(new /obj/item/storage/backpack/satchel/lockable/liaison(new_human), WEAR_BACK)
+	new_human.equip_to_slot_or_del(new /obj/item/reagent_container/glass/beaker/vial/random/good(new_human), WEAR_IN_BACK)
+	new_human.equip_to_slot_or_del(new /obj/item/stack/sheet/metal/med_small_stack(new_human), WEAR_IN_BACK)
+	add_random_cl_survivor_loot(new_human)
+	add_random_cl_survivor_loot(new_human)
```

Comment:
Did you mean to call this twice?

Reply:
> Did you mean to call this twice?

Yes, replaced the pistol (which I forgot I already gave him one) to yet another research loot spawner, didn't want a static so I put two there.